contracts/bonding/BondingManager.sol

```diff
@@ -163,40 +198,46 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
      * In both cases, a linear search will be executed starting at the hint to find the correct position. In the best case, the hint
      * is the correct position so no search is executed. See SortedDoublyLL.sol for details on list hints
      * @param _amount The amount of tokens to stake.
-     * @param _to The address of the transcoder to stake towards
-     * @param _onBehalfOf The address which will own the stake
+     * @param _orchestrator The address of the transcoder to stake towards
+     * @param _for The address which will own the stake
      * @param _oldDelegateNewPosPrev The address of the previous transcoder in the pool for the old delegate
      * @param _oldDelegateNewPosNext The address of the next transcoder in the pool for the old delegate
      * @param _currDelegateNewPosPrev The address of the previous transcoder in the pool for the current delegate
      * @param _currDelegateNewPosNext The address of the next transcoder in the pool for the current delegate
      */
-    function bondWithHint(
+    function bond(
```

Comment:
It is convention to prefix internal functions with an underscore ; i.e `_bond`

Tip: another convention to avoid _variable_ naming collisions is adding a trailing underscore